streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -233,7 +287,19 @@ private void runJoinBenchmark(final KafkaStreams streams, final String nameOfBen
 
 
 
-    public void processStream(String topic) {
+    public void processStream(String topic) throws Exception {
+        // see if we're in the load phase
+        if (loadPhase) {
```

Comment:
Same here - this block looks the same as other blocks of code. Perhaps extract into a method used by all